### PR TITLE
Add support for Lerkink Fan Switch

### PR DESCRIPTION
**Pull request by isakharov**
Add support for [Lerkink Fan Switch](https://aliexpress.ru/item/1005004705097813.html)
Add new tuyaTz datapoints key: fan_speed

### CODE REVIEW

File: devices/tuya.js [hunk 1]
```diff
@@ -4114,4 +4114,32 @@ module.exports = [
...
+        description: 'Fan switch',
+        fromZigbee: [tuya.fz.datapoints],
+        toZigbee: [tuya.tz.datapoints],
+        configure: tuya.configureMagicPacket,
+        exposes: [
+            tuya.exposes.switch(),
+            tuya.exposes.countdown().withValueMin(0).withValueMax(43200).withUnit('s').withDescription('Max ON time in seconds'),
+            exposes.numeric('fan_speed', ea.STATE_SET).withValueMin(1).withValueMax(5).withValueStep(1)
+                .withDescription('Speed off the fan'),
+            exposes.enum('power_on_behavior', ea.STATE_SET, ['off', 'on']).withDescription('Relay state'),
```
Comment (Koenkk) [inline devices/tuya.js:4130]: Can you confirm this is the Relay state after power recovery?

Comment (isakharov) [reply devices/tuya.js:4130]: Yep, it is a state after power recovery

**Comment by Koenkk** [comment]
Note to self; after this merge https://github.com/Koenkk/zigbee2mqtt.io/pull/1919

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**